src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
```

Comment:
dailyRent can also include days, so you don't have to use days in your if statements!